### PR TITLE
fix example message formatting

### PR DESCRIPTION
**Pull request by voidlock**
Updates the syslog examples to accurately reflect the syslog format logplex expects on ingestion

### CODE REVIEW

**Comment by ferd** [comment]
+1

**Comment by archaelus** [comment]
Did you regenerate these messages programmatically or edit them by hand? If by hand then the byte count of each message increased by 2, so the 130 and 134 byte counts will be incorrect in the modified messages right?

File: doc/README.http_input.md [hunk 1]
```diff
@@ -80,4 +80,4 @@ The following is the text of a POST to https://logplex.example.com using the tok
     User-Agent: Logsplat/0.1.0
     Content-Length: 1341
```
Comment (archaelus) [inline doc/README.http_input.md]: Also, this content length will now be wrong given the change in message sizes.

**Comment by agnaite** [comment]
closing as part of the [Retire Logplex OKR](https://salesforce.quip.com/UfD4AwLYWZzW).

**Pull request closed without merging**